src/python/pants/rules/core/strip_source_roots_test.py

```diff
@@ -86,7 +97,54 @@ def get_stripped_files_for_snapshot(
             get_stripped_files_for_snapshot(file_names, use_representative_path=False)
         ) == sorted(["project/example.py", "com/project/example.java"])
 
-    def test_strip_target(self) -> None:
+    def test_strip_sources_field(self) -> None:
```

Comment:
These tests are almost entirely copy-pastad from what we had before for testing `StripTargetRequest`, modulo differences to get it working with `SourcesField` instead of `TargetAdaptor`.